docs/guides/using-requirements-file.rst

```diff
@@ -0,0 +1,68 @@
+Using a Requirements File
+=========================
+
+The requirements file option is useful for specifying dependencies required for building the documentation. Additional uses specific to Read the Docs are mentioned at the end of this guide.
+
+For details about the purpose of pip requirements file and how to create one, check out `pip user guide`_  
+
+.. note:: For the purpose of building your documentation with RTD, *project* is the documentation project, and *project root* is the directory where all the documentation is stored, often named ``docs``. 
+
+To use the requirements file, create and place the requirements file in the root directory of your documentation directory. For example::
+
+    docs/requirements.txt
+
+Specifying a requirements file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Using the YAML configuration file
+---------------------------------
+The recommended approach for specifying a pip requirements file is to use a YAML (.yml) file. 
+
+See :doc:`yaml-config` for setting up the .yml file
+
+The requirements file can be specified here in the ``conda`` block. 
+The file's path should be relative to documentation root.
+
+.. code-block:: yaml
+
+	conda:
+	    requirements_file: requirements.txt
+
+Using the project admin dashboard
+---------------------------------
+
+Once the requirements file has been created;
+
+- Login to Read the Docs and go to the project admin dashboard.
+- Go to ``Admin > Advanced Settings > Requirements file``.
+- Specify the path of the requirements file you just created. The path should be relative to the root directory of the documentation project.
+
+Using requirements file with conda
+----------------------------------
+If using conda, its requirements file ``environment.yml`` can be specified in Read the Docs' YAML config file.
+
+More on Read the Doc's conda support: :doc:`conda`
+
+Working with `conda and environment.yml`_
+
+.. note:: Conda is only supported via the YAML file.
+
+The conda requirements file can be specified in ``readthedocs.yml`` in the ``conda`` block. 
+
+.. code-block:: yaml
+
+	conda:
+	    file: environment.yml
+
+As before, the path should be relative to the documentation repository root.
+
+Additional Uses
+~~~~~~~~~~~~~~~
+ - Documenting multiple packages
+ - setup.py not in root directory
+
+See :ref:`faq_document_package_not_at_root`
```

Comment:
Perhaps move this link to a prose block where it's more relevant? For example, "If you are noticing errors that Read the Docs can't find your requirements file, see :ref:`...`"

Reply:
I wrote this text for the bullet points as anyone to whom the condition is applicable will recognize  that there may be more info pertinent for them in this link.  
I didn't write the guide with the impression that these ref help solve the condition `when RTD cannot find the requirement`. Where can I understand this better?  
Due to the above, I am uncertain about where & how I can relocate this.  
Suggestions?